code/scripting/api/objs/enums.cpp

```diff
@@ -116,6 +116,9 @@ flag_def_list Enumerations[] = {
 	{"OS_SUBSYS_DEAD", OS_SUBSYS_DEAD, 0},
 	{"OS_SUBSYS_DAMAGED", OS_SUBSYS_DAMAGED, 0},
 	{"OS_SUBSYS_ROTATION", OS_SUBSYS_ROTATION, 0},
+	{"OS_PLAY_ON_PLAYER", OS_PLAY_ON_PLAYER, 0},
+	{"OS_LOOPING_DISABLED", OS_LOOPING_DISABLED, 0},
+	{"OS_STARTED_PLAYING", OS_STARTED_PLAYING, 0},
```

Comment:
I would remove `OS_STARTED_PLAYING` from this list since it's an internal flag.  Also remove `OS_USED` for the same reason, and remove `OS_DS3D` from both this list and the #define list since it's not used anywhere at all.